source/RialtoGStreamerMSEAudioSink.cpp

```diff
@@ -76,7 +79,7 @@ static GstStateChangeReturn rialto_mse_audio_sink_change_state(GstElement *eleme
         }
         if (priv->isVolumeQueued)
         {
-            client->setVolume(priv->volume);
+            client->setVolume(priv->targetVolume, priv->volumeDuration, priv->easeType);
```

Comment:
What happens if we have an audio-fade followed by a set volume from the client? Its going to send the previous values of duration and easeType from the audio-fade, so use defaults for these variables.

Reply:
I dont think you fixed this?